spec/features/users/sign_in_spec.rb

```diff
@@ -578,8 +578,8 @@
         to have_current_path(login_two_factor_path(otp_delivery_preference: 'sms', reauthn: false))
       expect(page).to have_content t(
         'two_factor_authentication.otp_delivery_preference.phone_unsupported',
-        location: 'Algeria',
-      )
+        location: '',
+      ).gsub(/\s+/, ' ')
       expect(user.reload.otp_delivery_preference).to eq 'voice'
```

Comment:
So since support for Algeria was removed, now the string "Algeria" isn't in our codebase...

Now that we have separate YML files, we could make a YML that just has all the country codes in the world and some default names?

Without a country name, the error message reads a little strangely:
> We’re unable to make phone calls to people in at this time.